packages/react/src/domain-config/ConfigView.tsx

```diff
@@ -29,6 +31,50 @@ function ConfigView() {
         }
     };
 
+    //Related to the ENS name input field
+    const [ensName, setEnsName] = useState<string | null>();
+    const [isValidEnsName, setIsValidEnsName] = useState<boolean>(true);
+    const [publishButtonState, setPublishButtonState] = useState<ButtonState>(
+        ButtonState.Idel,
+    );
+    const getEnsName = async () => {
+        if (state.connection.ethAddress && state.connection.provider) {
+            const name = await state.connection.provider.lookupAddress(
+                state.connection.ethAddress,
+            );
+            setEnsName(name);
+        }
+    };
+
+    const handleInputEnsName = async (
+        event: React.FormEvent<HTMLInputElement>,
+    ) => {
+        const ensName = (event.target as any).value;
+        setEnsName(ensName);
+        const isValidEnsName = ethers.utils.isValidName(ensName);
+        if (!isValidEnsName) {
+            setIsValidEnsName(false);
+            return;
+        }
+        const address = await state.connection.provider!.resolveName(ensName);
```

Comment:
Here you are getting the ETH record but I think you need to get the controller of the domain. Because the controller is the one that is able to set the records. Often the controller and the ETH record are the same but that is not always the case.